src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Colors.Net;
+using Newtonsoft.Json;
+using Azure.Functions.Cli.Interfaces;
+using DurableTask.Core;
+using DurableTask.AzureStorage;
+using static Colors.Net.StringStaticMethods;
+using DurableTask.Core.History;
+using Newtonsoft.Json.Linq;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager
+    {
+        private readonly ISecretsManager _secretsManager;
+        readonly AzureStorageOrchestrationService orchestrationService;
+        readonly TaskHubClient client;
+
+        public DurableManager(ISecretsManager secretsManager)
+        {
+            _secretsManager = secretsManager;
+
+            var connectionString = secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals("AzureWebJobsStorage", StringComparison.OrdinalIgnoreCase)).Value;
+
+            var settings = new AzureStorageOrchestrationServiceSettings
+            {
+                TaskHubName = "DurableFunctionsHub",
+                StorageConnectionString = connectionString,
+            };
+
+            this.orchestrationService = new AzureStorageOrchestrationService(settings);
+            this.client = new TaskHubClient(orchestrationService);
+        }
+
+        public async Task StartNew(string functionName, string version, string instanceId, object input)
+        {
+            await client.CreateOrchestrationInstanceAsync(functionName, version, instanceId, input);
+            var status = await client.GetOrchestrationStateAsync(instanceId, false);
+            if (status != null)
+            {
+                ColoredConsole.WriteLine(Yellow($"Started {status[0].Name} with new instance {status[0].OrchestrationInstance} at {status[0].CreatedTime}"));
+            }
+            else
+            {
+                ColoredConsole.WriteLine(Red($"Could not start new instance {instanceId}"));
```

Comment:
If this is an error, then throw a 
```csharp
new CliException($"Could not start new instance {instanceId}");
```
 that way the we can set the exit code of the process to indicate an error.